crates/nu-command/src/path/exists.rs

```diff
@@ -152,4 +170,26 @@ mod tests {
 
         test_examples(SubCommand {})
     }
+
+    #[test]
+    fn test_check_symlink_exists() {
```

Comment:
It's not a right place for adding test which required with `nu!` macro, it's better to add to `crates/nu-command/tests/commands/path/exists.rs`